xrootd/launch_linux.go

```diff
@@ -95,6 +95,10 @@ func (plauncher PrivilegedXrootdLauncher) Launch(ctx context.Context) (context.C
 	if plauncher.fds[1] != -1 {
 		env = []string{"XRDHTTP_PELICAN_INFO_FD=3"}
 	}
+	// Set the directory where the SciTokens library should cache public keys.
+	if xdg_cache_home, ok := os.LookupEnv("XDG_CACHE_HOME"); ok {
```

Comment:
Can you switch `xdg_cache_home` to `xdgCacheHome` to stay consistent with our use of camel case?